scripts/deploy-kubevirt-and-cdi.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+
+KUBEVIRT_NIGHTLY=$(curl -L https://storage.googleapis.com/kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/latest)
```

Comment:
Can you change it to `KUBEVIRT_VERSION` and make it use the latest stable version by default? This will help pinning `KUBEVIRT_VERSION` to a specific value in release branches.

Reply:
Done